src/nrc/setup_configuration/models.py

```diff
@@ -81,3 +86,10 @@ class Meta:
 
 class AbonnementConfigurationModel(ConfigurationModel):
     items: list[AbonnementConfigurationItem]
+
+
+class NotificationConfigurationModel(_NotificationConfigurationModel):
+    notifications_api_service_identifier: Optional[str] = Field(default=None)
```

Comment:
Let's also add an example value
```suggestion
    notifications_api_service_identifier: Optional[str] = Field(default=None, examples=["notificaties-api"])
```